external/anomaly/ote_anomalib/exportable_code/anomaly_classification.py

```diff
@@ -0,0 +1,59 @@
+"""Wrapper for Open Model Zoo for Anomaly tasks."""
+
+# INTEL CONFIDENTIAL
+#
+# Copyright (C) 2021 Intel Corporation
+#
+# This software and the related documents are Intel copyrighted materials, and
+# your use of them is governed by the express license under which they were provided to
+# you ("License"). Unless the License provides otherwise, you may not use, modify, copy,
+# publish, distribute, disclose or transmit this software or the related documents
+# without Intel's prior written permission.
+#
+# This software and the related documents are provided as is,
+# with no express or implied warranties, other than those that are expressly stated
+# in the License.
+
+from typing import Any, Dict
+
+import cv2
+import numpy as np
+from openvino.model_zoo.model_api.models import SegmentationModel
+from openvino.model_zoo.model_api.models.types import NumericalValue
+
+
+class AnomalyClassification(SegmentationModel):
+    """Wrapper for anomaly classification task."""
+
+    __model__ = "anomaly_classification"
+
+    @classmethod
+    def parameters(cls):
+        """Dictionary containing model parameters."""
+        parameters = super().parameters()
+        parameters["resize_type"].update_default_value("standard")
+        parameters.update(
+            {
+                "threshold": NumericalValue(default_value=20, description="Threshold value to locate anomaly"),
```

Comment:
@akorobeinikov @ashwinvaidya17 @samet-akcay @djdameln 
this should be fixed as soon as confidence normalization issue is fixed

Reply:
fixed

Reply:
your recent commit has been removed, because it has to be checked in CI, we can't wait more time. Looks like it should be fixed together with fix of confidence normalization. 
If you think that it should be fixed right now please create a separate PR.